### DISCUSSION TITLE
Couldn't sign up, didn't receive any sms verification

### DISCUSSION THREAD

**Discussion by nskhoa1502** [vercel/community#4175] [Help] [answered]
### Summary

Hi, I couldn't able to sign up to Vercel becasue there are no sms verfication sent to my mobile phone. Is there any thing i can do ?

### Steps to Reproduce

Hi, I couldn't able to sign up to Vercel becasue there are no sms verfication sent to my mobile phone. Is there anything i can do ?

**Comment by swarnava** [comment, marked as answer]
Hi,

Please email at registration@vercel.com
